### PR TITLE
Feature add entry

### PR DESCRIPTION
**Pull request by Kajochi**

### CODE REVIEW

File: backend/src/main/java/de/neuefische/capstone/backend/entries/EntriesService.java [hunk 1]
```diff
@@ -9,11 +10,26 @@ public class EntriesService {
...
+        return entriesRepo.save(new Entry(
+                idService.createRandomId(),
+                entryWithNoId.getTitle(),
+                entryWithNoId.getDescription(),
+                entryWithNoId.getDate(),
+                entryWithNoId.getAmount(),
+                entryWithNoId.getCategory(),
+                entryWithNoId.getInterval()
+        ));
+    }
```
Comment (cananozyol) [inline backend/src/main/java/de/neuefische/capstone/backend/entries/EntriesService.java:34]: just out of curiosity why did you use the save and not insert method here?

File: backend/src/test/java/de/neuefische/capstone/backend/entries/EntriesIntegrationTest.java [hunk 2]
```diff
@@ -34,5 +34,31 @@ void WhenListIsEmptyReturnEmptyList() throws Exception {
...
+        //Then
+                .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+                .andExpect(jsonPath("$.title").value("testTitle"))
+                .andExpect(jsonPath("$.description").value("testDescription"))
+                .andExpect(jsonPath("$.date").value("2023-12-03"))
+                .andExpect(jsonPath("$.amount").value(34))
+                .andExpect(jsonPath("$.category").value("INCOME"))
+                .andExpect(jsonPath("$.interval").value("MONTHLY"));
+    }
+
```
Comment (cananozyol) [inline backend/src/test/java/de/neuefische/capstone/backend/entries/EntriesIntegrationTest.java:62]: if you like you can also add this line to your test to check that you're getting the expected status(200):

.andExpect(MockMvcResultMatchers.status().isOk())

**Review by cananozyol — APPROVED** [review]
Clean Code 💫 even in the massive EntryAddUpdate component 
LGTM 🚀

File: backend/src/main/java/de/neuefische/capstone/backend/entries/EntriesService.java (hunk 1 above)
Comment (Kajochi) [reply backend/src/main/java/de/neuefische/capstone/backend/entries/EntriesService.java:34]: Thats a really good point. I should use insert here. Thanks:D

File: backend/src/test/java/de/neuefische/capstone/backend/entries/EntriesIntegrationTest.java (hunk 2 above)
Comment (Kajochi) [reply backend/src/test/java/de/neuefische/capstone/backend/entries/EntriesIntegrationTest.java:62]: Oh yes i will add that too

**Pull request merged**